### ISSUE TITLE
fails to get epub from lightnovels.me

### ISSUE THREAD

**Issue by mapreri** [dteviot/WebToEpub#855]
**Describe the bug**

I was trying to get novels out of Lightnovels.me, but got an error instead

**To Reproduce**
1. Go to https://lightnovels.me/novel/illicit-relationship-7590.html
2. Click on the extension button
3. Click on "Pack epub"

```
Error: Fetch of URL 'https://pandapama.com/read/illicit-relationship/chapter-2-teasing-lin-xing-xue-793-1634639668933.html' failed with network error Failed to fetch.
    at FetchErrorHandler.onFetchError (chrome-extension://akiljllkbielkidmammnifcnibaigelm/js/HttpClient.js:28:31)
    at chrome-extension://akiljllkbielkidmammnifcnibaigelm/js/HttpClient.js:158:45
    at async LightnovelsmeParser.fetchChapter (chrome-extension://akiljllkbielkidmammnifcnibaigelm/js/Parser.js:543:17)
    at async Promise.all (index 0)
    at async LightnovelsmeParser.fetchWebPages (chrome-extension://akiljllkbielkidmammnifcnibaigelm/js/Parser.js:470:17)
```

Note that lightnovels.me uses pandapama.com for storing and serving the files.

**Desktop (please complete the following information):**
 - OS: Debian (unstable, fwiw)
 - Browser chromium 107
 - Version extension 0.0.0.150

Thank you for your nice project!!

**Comment by dteviot** [comment]
@mapreri 
A "Failed to fetch" network error usually means the browser was unable to contact the requested server.
So, a network connectivity error, or server offline.
Site seems to be up now.

**Comment by mapreri** [comment]
You are right, howver now trying it I get:

```
Error: Could not find content element for web page 'https://lightnovels.me/novel/illicit-relationship-7590.html'.
    at chrome-extension://akiljllkbielkidmammnifcnibaigelm/js/Parser.js:500:23
    at async Promise.all (index 0)
    at async LightnovelsmeParser.fetchWebPages (chrome-extension://akiljllkbielkidmammnifcnibaigelm/js/Parser.js:470:17)
```

which is probably more interesting?

**Comment by dteviot** [comment]
@mapreri 
https://lightnovels.me/novel/illicit-relationship-7590.html isn't the URL of a chapter.  
I'm guessing "illicit relationship" is first item in the list of chapters to collect?  I'd suggest you remove it.

**Comment by mapreri** [comment]
mh, good point.  I figured that something was clever enough to use that as an index or something, but indeed it doesn't make much sense.

So, removing that I get that network error I posted at the start.

Now, I had a look at chrome's console, and I see what's the actual error:

```
Access to fetch at 'https://pandapama.com/read/illicit-relationship/chapter-1-new-life-625-1634639668931.html' from origin 'chrome-extension://akiljllkbielkidmammnifcnibaigelm' has been blocked by CORS policy: The value of the 'Access-Control-Allow-Origin' header in the response must not be the wildcard '*' when the request's credentials mode is 'include'.
```

with that in mind, I tried running chromium with the flags `--disable-site-isolation-trials --disable-web-security` so to disable CORS' checks, and that seems to be working (it's crunching through the chapters without errors for now, at least).

Quite interestingly, when running webtoepub in that temporary chromium profile with --disable-web-security I also don't get shown https://lightnovels.me/novel/illicit-relationship-7590.html  in the list of chapters like instead I'm being proposed here, I wonder why...

So, I reckon this might be my actual error perhaps?
